src/set_theory/pgame.lean

```diff
@@ -600,6 +600,13 @@ end
   move_right (-x) ((right_moves_neg x).symm i) = -(move_left x i) :=
 by { cases x, refl }
 
+/-- If `x` has the same moves as `y`, then `-x` has the sames moves as `-y`. -/
+def neg_congr_relabelling : ∀ {x y : pgame}, x.relabelling y → (-x).relabelling (-y)
+| (mk xl xr xL xR) (mk yl yr yL yR) ⟨L_equiv, R_equiv, L_relabelling, R_relabelling⟩ :=
+  ⟨R_equiv, L_equiv,
+    λ i, neg_congr_relabelling (by simpa using R_relabelling (R_equiv i)),
+    λ i, neg_congr_relabelling (by simpa using L_relabelling (L_equiv.symm i))⟩
```

Comment:
I think you should rename this to enable dot notation:
```suggestion
/-- If `x` has the same moves as `y`, then `-x` has the sames moves as `-y`. -/
def relabelling.neg_congr : ∀ {x y : pgame}, x.relabelling y → (-x).relabelling (-y)
| (mk xl xr xL xR) (mk yl yr yL yR) ⟨L_equiv, R_equiv, L_relabelling, R_relabelling⟩ :=
  ⟨R_equiv, L_equiv,
    λ i, relabelling.neg_congr (by simpa using R_relabelling (R_equiv i)),
    λ i, relabelling.neg_congr (by simpa using L_relabelling (L_equiv.symm i))⟩
```

Reply:
Cool, I'll fix this. 

While I'm at it I'll also change the `equiv_of_relabelling`